test/classes/test_simplicial_complex.py

```diff
@@ -167,7 +168,14 @@ def test_setitem__(self):
         SC.add_simplex(s)
         assert SC["A"]["heat"] == 1
 
-    def add_simplices_from(self):
+    def test_maxdim(self):
+        """Test deprecated maxdim property for deprecation warning."""
+        with pytest.deprecated_call():
+            # Cause a warning by accessing the deprecated maxdim property
+            SC = SimplicialComplex()
+            _ = SC.maxdim
```

Comment:
`SimplicialComplex.maxdim` should not only raise a deprecation warning but still also return the correct value. Maybe something like this would be better?

```python
with pytest.deprecated_call():
    SC = SimplicialComplex([(1, 2, 3)])
    assert SC.maxdim == 2
```

Reply:
Missed to add this back! Will do